go.work

```diff
@@ -11,3 +11,5 @@ use ./x/ibc-hooks
 use ./x/epochs
 
 replace github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.16.5
+
+replace github.com/cosmos/cosmos-sdk => github.com/osmosis-labs/cosmos-sdk v0.50.14-v30-osmo
```

Comment:
Should we bump the version in the go.mod in each submodule or do you think this is ok?

Reply:
Oh yeah I missed them, I think we should for consistency sake.